idd/Energy+.idd.in

```diff
@@ -5117,7 +5117,6 @@ WindowMaterial:SimpleGlazingSystem,
        \note Note that the effective upper limit for U-factor is 5.8 W/m2-K
        \units W/m2-K
        \minimum> 0
-       \maximum  7
```

Comment:
I strongly suspect there is still an upper limit, it is just higher than 7.  I would suggest doing some trial and error testing to find what the new upper limit is and change the value here, rather than remove this.

Reply:
> > Something seems off in the reporting or presentation of convection coefficients. I would expect the Hc values to go up with the change here, sometimes by a lot. But the summary shows them going down after adjustment.
> 
> Hi @EnergyArchmage, I have forgotten to adjust the radiative conductance terms in the heat balance equation. This might have affected the convective coefficient computation. I am fixing it now. I'll keep you updated on the changes.

Hi @EnergyArchmage, I updated the table and the comparison plot of convection coefficients, surface temperature, and zone temperature. Adding adjustments to the radiative coefficients (hr(i) terms in heat balance equations) seems to lead to large changes in the five terms above.

Reply:
Are there some specific signs making it seems there is still some higher-than-7 upper limit?

Reply:
Or do you mean we need to alter this idd field value from 7 to some higher value so that users will not input unreasonably large U values (like 50)?

Reply:
@EnergyArchmage Current implementation allows adjustment ratio to be any number, given the user input. If a user inputs a crazily large U value, the adjustment ratio would just be calculated accordingly to be used in the heat balance equation. We will test with some very large inputs to see if the equations are still able to be solved and if any run time error would occur.

Reply:
@EnergyArchmage I tried out a set of different input U values using the RefBldgWarehouseNew2004_Chicago.idf model. Following are plots of a few key parameters when with various input U values. All tests can run through, but some of the parameters don't seem very reasonable. For example, large U's will result in negative inside surface temperature in summer.
![hcout_varyU](https://user-images.githubusercontent.com/13421161/129984759-dc57bbcf-c635-44fd-8b43-41f46fc506ad.png)
![hcin_varyU](https://user-images.githubusercontent.com/13421161/129984775-271b24b0-51c7-4ee5-a823-50fb12c8bab8.png)
![thetas_1_varyU](https://user-images.githubusercontent.com/13421161/129984776-f06c6b74-b7df-47c5-bfd5-9f2a30001602.png)
![thetas_2_varyU](https://user-images.githubusercontent.com/13421161/129984782-db6e2f21-d5fd-4e2a-bce4-de233bdea584.png)
![mat_3_varyU](https://user-images.githubusercontent.com/13421161/129984786-854469ba-c111-480b-93cc-ffd133415e79.png)